src/book/02-projects/04-other-projects/02-nupbr.mdx

```diff
@@ -0,0 +1,240 @@
+This includes:
+
+| Config Parameter | Type    | Example Value | Description                                                                                                                                |
+| ---------------- | ------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `num_images`     | `int`   | `10`          | Number of generated image sets (where a raw image, segmentation mask and depth image will be generated for each camera)                    |
+| `output_stereo`  | `bool`  | `True`        | Determines if stereo images will be generated, separated by `0.5 * camera["stereo_camera_distance"]` (configured in `pbr/scene_config.py`) |
+| `output_depth`   | `bool`  | `True`        | Determines if depth images will be generated.                                                                                              |
+| `output_base`    | `str`   | `"./run_{}"`  | Defines the output directory format string to allow for new output directory creation on regeneration.                                     |
+| `filename_len`   | `int`   | `10`          | Length of the output file, e.g. for `filename_len = 10`, the first raw image generated will be saved as `0000000000.jpg`.                  |
+| `image_dirname`  | `str`   | `"raw"`       | Directory name in which raw images are saved. e.g. `./run_0/raw`.                                                                          |
+| `mask_dirname`   | `str`   | `"seg"`       | Directory name in which segmentation are saved. e.g. `./run_0/seg`.                                                                        |
+| `depth_dirname`  | `str`   | `"depth"`     | Directory name in which depth images are saved. e.g. `./run_0/depth`.                                                                      |
+| `meta_dirname`   | `str`   | `"meta"`      | Directory name in which the metadata for each image set is saved. e.g. `./run_0/meta`.                                                     |
+| `max_depth`      | `float` | `20`          | Maximum depth for the normalised depth map (in metres).                                                                                    |
+
+Once the parameters are set, NUpbr can be run using
+
+```bash
+# Show Blender GUI
+blender --python pbr.py
+
+# Headless
+blender --python pbr.py -b
+```
+
+---
+
+## Status
+
+#### Ongoing Work
+
+- Rigging of robot models to generate realistic poses
+- Replacement of hair particle system with grass cards to reduce performance overhead
+
+#### Future Work
+
+- Porting to Blender 2.8
+- Porting field UV map generation to OpenCV
+  ng to Blender 2.8
```

Comment:
I'm not sure what's going on with these lines, I think you were meaning to have another bullet like
```suggestion
  - Update to Blender 2.8
```
?